pkgs/by-name/la/labwc-tweaks/package.nix

```diff
@@ -1,44 +1,42 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, meson
-, ninja
+, cmake
+, perl
 , pkg-config
-, gtk3
-, libxml2
+, qt6
 , xkeyboard_config
-, wrapGAppsHook
 , unstableGitUpdater
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation {
   pname = "labwc-tweaks";
-  version = "unstable-2024-04-02";
+  version = "unstable-2024-04-25";
```

Comment:
```suggestion
  version = "0-unstable-2024-04-25";
```

Reply:
It can be done, but it seems that the update script will remove the "0-" prefix when generating  update PRs automatically.

Reply:
ah, ignore me then, I only post this suggestion to manually updated packages